app/controllers/solicitations_controller.rb

```diff
@@ -13,6 +13,11 @@ def create
     @partial = 'solicitations/thank_you'
     CompanyMailer.confirmation_solicitation(@solicitation.email).deliver_later
     AdminMailer.solicitation(@solicitation).deliver_later
+
+    respond_to do |format|
+      format.html { redirect_to (landing_path(params[:solicitation][:form_info][:slug]) + "#section-formulaire"), notice: t('.thanks') }
```

Comment:
On peut l’écrire comme ça :

```suggestion
      format.html { redirect_to (landing_path(@solicitation.slug, anchor: "section-formulaire"), notice: t('.thanks') }
```